README.md

```diff
@@ -83,10 +83,14 @@ Subdir `vscode-client` contains the sources of a VSC Extension; a LSP client tha
 
 ## Build
 
+> HINT: On debian, installing `npm` from repositories (`sudo apt install npm`) will install a version 5.x of `npm`, incompatible with Node.js 10.x.
```

Comment:
I'd probably say that it is likely to be 'outdated', instead of explaining that some specific version is not compatible with some other. On the one hand, it's make it easier to keep this readme up to date. On the other hand, I'm not sure about that warning being the real source of the issues.

Reply:
> will probably install an outdated version of `npm` with regard to Node.js version

Would that be better ?

Reply:
Go ahead ;)